src/main/java/com/salesforce/mirus/metrics/MirrorJmxReporter.java

```diff
@@ -112,6 +147,17 @@ public synchronized void recordMirrorLatency(String topic, long millis) {
     if (sensor != null) {
       sensor.record((double) millis);
     }
+
+    Map<Integer, Sensor> bucketSensors = histogramLatencySensors.get(topic);
+    bucketSensors
+        .entrySet()
+        .stream()
+        .forEach(
+            sensorEntry -> {
+              if (millis > sensorEntry.getKey()) {
```

Comment:
I think this should this be `>=` so we are guaranteed to catch all records in the "0m" bucket.

Reply:
Actually, I think you can simplify a bit here:
```
bucketSensors
            .forEach((edgeMillis, bucket) -> {
              if (millis >= edgeMillis) {
                bucket.record(1);
              }
            });

```